.github/CODEOWNERS

```diff
@@ -0,0 +1,16 @@
+# Documentation/Infrastructure
+/docs										@justinahinon @felixarntz	
+
+# Module: Images
```

Comment:
"Images" as well as the other ones below are rather our focuses, not modules.
```suggestion
# Focus: Images
```

Could you change that below accordingly?